quotes/ContentView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ContentView.swift
+//  quotes
+//
+//  Created by Jason Zurita on 4/05/21.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    @State var quote: Quote?
+    var body: some View {
+        VStack {
+            if let q = quote {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/swift-linting-formatting-workshop/blob/2f007754112214b4f5103cca96bc3418eef48e0d/quotes/ContentView.swift#L14"></span>Variable name should be between 3 and 40 characters long: 'q'<br />
<code>identifier_name</code> <code>ContentView.swift:14</code></td>
    </tr>
  </tbody>
</table>